src/components/media/MuiAvatarShim.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const VALID_SIZES = ['s', 'm', 'l', 'xl'];
+
+/**
+ * @module MuiAvatarShim
+ * DO NOT DOCUMENT (not for public use)
+ *
+ * This shim replaces material-ui Avatar to cover usage in FDS.
+ * Renders a basic avatar.
+ */
+const MuiAvatarShim = ({ name, src, size, className, style }) => {
+  const classNames = [
+    'fdsAvatar',
+    'alignChild--center--center',
+    `media--${size}`,
+    className,
+  ].join(' ');
```

Comment:
This is just a shim, so not sure how closely I should be looking at these things 😄but wondering why you opted to go with array.join here vs classnames. I feel like the shim itself is temporary but assuming it wasn't, I feel like using the array with class join will encourage people to keep using it, even when they do need a classname in this list to be conditionally rendered, and then they'll just array.push it in.

Reply:
I didn't think `classnames` was necessary here, but you're right, we don't want this pattern to spread. I'll update it